charts/telegraf/templates/deployment.yaml

```diff
@@ -39,6 +39,25 @@ spec:
         - name: {{ .name }}
           mountPath: {{ .mountPath }}
         {{- end }}
+        {{- if .Values.hooks }}
```

Comment:
This would lead to a failed deployment if someone has `hooks: {}` in their values.yaml.

Could we maybe use `{{- if or (.Values.hooks.preStop) (.Values.hooks.postStart) }}` ?

I've not tested this, but I think it might work :)

Reply:
Good call! Pushed the change 👍